src/main/java/games/strategy/triplea/printgenerator/PrintGenerationData.java

```diff
@@ -1,14 +1,12 @@
 package games.strategy.triplea.printgenerator;
 
 import java.io.File;
-import java.util.Map;
 
 import games.strategy.engine.data.GameData;
 
 public class PrintGenerationData {
-  private File m_outDir;
-  private Map<Integer, Integer> m_SamePUMap;
```

Comment:
The accessor and mutator for `m_SamePUMap` were never called, so I just removed this field.